src/database/Database.cpp

```diff
@@ -212,31 +212,6 @@ Database::applySchemaUpgrade(unsigned long vers)
     soci::transaction tx(mSession);
     switch (vers)
     {
-    case 14:
-        mApp.getPersistentState().setRebuildForType(OFFER);
-        break;
-    case 15:
-        mApp.getPersistentState().setRebuildForType(TRUSTLINE);
-        mApp.getPersistentState().setRebuildForType(LIQUIDITY_POOL);
-        break;
-    case 16:
-        mApp.getPersistentState().setRebuildForType(LIQUIDITY_POOL);
-        break;
-    case 17:
-        mApp.getPersistentState().setRebuildForType(OFFER);
-        break;
-    case 18:
-        createTxSetHistoryTable(*this);
-        mApp.getPersistentState().upgradeSCPDataFormat();
-        break;
-    case 19:
-        mApp.getPersistentState().upgradeSCPDataV1Format();
-        break;
-    case 20:
-        mApp.getPersistentState().setRebuildForType(CONFIG_SETTING);
-        mApp.getPersistentState().setRebuildForType(CONTRACT_DATA);
-        mApp.getPersistentState().setRebuildForType(CONTRACT_CODE);
-        break;
```

Comment:
`case 21:` can also be removed now that `MIN_SCHEMA_VERSION` is 21.

Reply:
Removed. I preserved the switch structure for future upgrades.

Reply:
Or should I just delete it all?

Reply:
I think preserving the switch is good. Can you please add a comment about what should be done on a schema version upgrade and why this is empty at the moment?